scss/_objects.grid.scss

```diff
@@ -4,47 +4,76 @@
     Grid system objects, works together with utility width classes
 */
 
+
+// ---
+// Variables
+// ---
+
+$grid-max-width : map-get($breakpoints,bp5) !default;
+$grid-gutter    : $columns-gutter !default;
+
+
+// ---
+// Styles
+// ---
+
+
+// Max width container
 .o-container {
     margin-left: auto;
     margin-right: auto;
```

Comment:
I'm not a big fan of putting `margin: 0` on something when it is not really necessary. So the non-shorthand version makes more sense to me here.